website/docs/r/validating_webhook_configuration.html.markdown

```diff
@@ -0,0 +1,129 @@
+---
+layout: "kubernetes"
+page_title: "Kubernetes: validating_webhook_configuration"
+sidebar_current: "docs-kubernetes-resource-validating-webhook-configuration"
+description: |-
+  Validating Webhook Configuration automatically scales the number of pods in a replication controller, deployment or replica set based on observed CPU utilization.
+---
+
+# validating_webhook_configuration
+
+Validating Webhook Configuration configures a [validating admission webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks).
+
+## Example Usage
+
+```hcl
+resource "validating_webhook_configuration" "example" {
+  metadata {
+    name = "test.terraform.io"
+  }
+
+  webhook {
+    name = "test.terraform.io"
```

Comment:
I was unable to apply this configuration without specifying the `side_effect` attribute:
```suggestion
    name = "test.terraform.io"
    side_effects = "None"
```